src/main/groovy/com/vherasymenko/avro/decoder/core/CourseInstallDecoderService.groovy

```diff
@@ -1,23 +1,31 @@
 package com.vherasymenko.avro.decoder.core
 
 import com.vherasymenko.avro.decoder.core.json.AvroJsonDecoderPort
-import event.course_install.CourseInstall
+import groovy.util.logging.Slf4j
+import org.apache.avro.Schema
+import org.springframework.cloud.stream.schema.client.SchemaRegistryClient
 
 /**
  * Decoder for the course install document.
  */
+@Slf4j
```

Comment:
Why do you think it's necessary? The Lockback gives enough functionality we need for this project and is simple to use.